.travis.yml

```diff
@@ -8,7 +9,14 @@ env:
   - CF_SPACE="production"
   - CF_USERNAME="govuk-design-system-deploy-production@digital.cabinet-office.gov.uk"
   # CF_PASSWORD
```

Comment:
Does this comment need updated? The NPM_TOKEN environment variable is presumably set in here as well now?

Reply:
we set an NPM_TOKEN env variable on TRAVIS CI.